eZ/Publish/API/Repository/Tests/Values/User/Limitation/ObjectStateLimitationTest.php

```diff
@@ -144,4 +147,181 @@ public function testObjectStateLimitationForbid()
         $contentService->deleteContent($draft->contentInfo);
         /* END: Use Case */
     }
+
+    /**
+     * Tests an ObjectStateLimitation.
+     *
+     * @see \eZ\Publish\API\Repository\Values\User\Limitation\ObjectStateLimitation
+     *
+     * @throws \ErrorException
+     * @expectedException \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
```

Comment:
Nitpick: This test method is quite long and performs multiple API calls that might throw `UnauthorizedException`, so just to be safe I'd add here `@expectedExceptionMessage` containing relevant part of expected message. Even if right now it doesn't make difference, when some other code evolves we might not notice that `UnauthorizedException` starts referring to something else. Does this make sense? :)